CMakeLists.txt

```diff
@@ -46,11 +46,11 @@ CHECK_TYPE_SIZE(long SIZE_OF_LONG)
 CHECK_TYPE_SIZE(off_t SIZE_OF_OFF_T)
 
 if ("${SIZE_OF_OFF_T}" LESS 8)
-  option(CGNS_ENABLE_LFS "Enable or disable large file support" "OFF")
+  option(CGNS_ENABLE_LFS "Enable or disable large file support" "ON")
```

Comment:
Is this actually related to enabling 64 bit?  Based on documentation it is for large file support which is support for files larger than 2GB which can happen with or without the 64BIT being enabled or disabled.

Reply:
You are right this is not related directly to 64bit. It is just that most of the time when 64 bit is required, LFS will be needed. In fact as the cmake is currently written, it activates only on Windows OS as far as I know.

Reply:
Ref. https://cgnsorg.atlassian.net/browse/CGNS-184